src/libs/Environment/isDevelopment.js

```diff
@@ -0,0 +1,12 @@
+import lodashGet from 'lodash/get';
```

Comment:
I was trying to figure out why this was in it's own file, and then it dawned on me why you did this (and I totally see the logic). I'm not sure it's what we want though. This doesn't really fit our cross-platform pattern because there is a cross-platform module (`Environment`) that is sharing non-platform specific methods (`isDevelopment`).

I think rather we should have `Environment` be non-platform specific and then making only the pieces that need to be platform-specific (`getEnvironment` in this case) would be split into their platform files.

I hope that makes sense?